apps/studio/components/interfaces/Organization/Usage/Usage.constants.tsx

```diff
@@ -254,7 +254,7 @@ export const USAGE_CATEGORIES: CategoryMeta[] = [
         name: 'Edge Function Invocations',
         unit: 'absolute',
         description:
-          'Every serverless function invocation independent of response status is counted.\nBilling is based on the sum of all invocations throughout your billing period.',
+          'Every serverless function invocation independent of response status is counted, encompassing operations initiated by the HTTP verbs: GET, POST, PUT, PATCH, DELETE.\n Billing is based on the sum of all invocations throughout your billing period.',
```

Comment:
Probably more direct to say "OPTIONS requests do not count towards this metric"

Reply:
We also don't count `HEAD`. But I can flip to the negatives if you prefer it.

Reply:
> We also don't count HEAD

oh i didnt know that either. i think flipping it is definitely way more obvious then